src/Apps/Collect/__tests__/FilterState.test.tsx

```diff
@@ -55,6 +56,15 @@ describe("FilterState", () => {
       done()
     })
   })
+
+  it("returns a height range tuple based on filter string", done => {
+    instance.setFilter("height_range", "*-50", mediator)
+
+    setTimeout(() => {
+      expect(instance.heightRangeToTuple()).toEqual([1, 50])
+      done()
+    })
```

Comment:
Is `setTimeout` necessary? If so, and your not looking to "wait" for a specific amount of time, there's `setImmediate` which means "wait until the next run of the JS event loop" -- this is typically all that's needed. (But the better solution is to not use either unless totally necessary)

Reply:
Ah yup ^ https://github.com/artsy/reaction/blob/master/src/DevTools/enzyme%2BrenderUntil.d.ts

Forgot about this fellow.

Reply:
@orta and I took a look at this and made most of them synchronous; updated in the most recent commit. Will investigate `renderUntil` for the future though!